libnfc/buses/uart.c

```diff
@@ -367,6 +384,26 @@ uart_receive(serial_port sp, uint8_t *pbtRx, const size_t szRx, void *abort_p, i
   return NFC_SUCCESS;
 }
 
+/**
+ * @brief Send \a pbtTx content to UART one byte at a time
+ *
+ * @return 0 on success, otherwise a driver error is returned
+ */
+int
+uart_send_single(serial_port sp, const uint8_t *pbtTx, const size_t szTx, int timeout)
+{
+  (void) timeout;
+  int error = 0;
+  for (int i = 0; i < szTx; i++)
+  {
+    error |= uart_send(sp, pbtTx+i, 1, timeout);
+    usleep(9);
```

Comment:
Sleeping in userland seems terrible…  Timing should be managed at the underlying level (UART driver of the OS?)